src/Step/Step.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/no-unused-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Wizard Step wrapper
+ * @param children
+ * @param id
+ */
+const Step = ({ children, id }) => (
+  <div
+    id={id}
+  >
+    {children}
+  </div>
+);
+
+
+Step.propTypes = {
+  children: PropTypes.node,
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string,
+  nextBtnTitle: PropTypes.string,
+  prevBtnTitle: PropTypes.string,
+  onEnter: PropTypes.func,
+  onLeave: PropTypes.func,
+  enabled: PropTypes.bool,
```

Comment:
Till know its's only children and id .. please remove all non used proptype validations .. and update linter file to catch this